web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -131,6 +133,39 @@ export function ProjectCard({
       'https://schema.regen.network#additionalCertifications'
     ]?.['@list'];
 
+  const createPostButton = (
```

Comment:
it's not necessarily a "create post" button, as you can see in the `onClick` implementation, it could also be a "buy" or "view project" button so I would rename it and also extract it into another component so it doesn't remount this on every render